cli/command/user/resend-verify.go

```diff
@@ -0,0 +1,53 @@
+package user
+
+import (
+	"errors"
+
+	"github.com/appcelerator/amp/api/rpc/account"
+	"github.com/appcelerator/amp/api/rpc/version"
+	"github.com/appcelerator/amp/cli"
+	"github.com/appcelerator/amp/cmd/amplifier/server/configuration"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/status"
+)
+
+// NewResendVerifyCommand returns a new instance of the resend-verify command.
+func NewResendVerifyCommand(c cli.Interface) *cobra.Command {
+	return &cobra.Command{
+		Use:     "resend-verify USERNAME",
```

Comment:
instead of `resend-verify`, which sounds more like the second email (you have been verified), why not `resend-verification-token`. It's a bit long, but that makes things clearer.